app/exporters/patients_exporter.rb

```diff
@@ -116,9 +120,18 @@ def self.csv_fields(patient)
       patient.id,
       *medications_for(patient)
     ]
+
+    csv_fields.insert(zone_column_index, patient.address.zone) if zone_column_index
```

Comment:
Ugh, this makes me so uncomfortable, but I think it's the simplest way to do this 😅

Reply:
Same, it was a bit painful to write this 😅

Reply:
I imagined reworking this exporter to use some sort of smart data structure like:

```ruby
{
  zone: {
    header: "Patient Zone",
    extractor: ->(patient) { patient.address.zone }
  },
  ...
}
```

but seemed like too much indirection. Sticking the zone in there manually is very KISS.

Reply:
(don't tell anyone but I tried out a whole refactor using a hash map 😬  ended up deleting and came back to this)